e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts

```diff
@@ -238,6 +238,54 @@ class RegistrationDetails {
     ).toContain(userName);
     expect(await timeStampLocator.textContent()).toContain(date);
   }
+
+  async addNote(actions: string, addNote: string) {
+    await this.page.getByText(actions).click();
+    await this.page.getByText(addNote).click();
+  }
+
+  async writeNote(placeholder: string, note: string, buttonName: string) {
+    const okButton = this.page.getByRole('button', {
+      name: buttonName,
+    });
+    await this.page.getByPlaceholder(placeholder).fill(note);
+    await this.page.waitForLoadState('networkidle');
+    await expect(okButton).toBeEnabled();
```

Comment:
Would it be good to check `toBeDisabled` before you fill the note?

Reply:
Could be actually. It is a fair point it was not mentioned in the "feature file" but I think it is required for it to be disabled before the note being typed.

Reply:
Added